code/modules/antagonists/changeling/changeling.dm

```diff
@@ -504,6 +504,8 @@
 	new_profile.age = target.age
 	for(var/datum/quirk/target_quirk in target.quirks)
 		LAZYADD(new_profile.quirks, new target_quirk.type)
+	if(target.wear_id?.get_gun_permit_iconstate()=="hud_permit")
```

Comment:
I feel like `"hud_permit"` should be a define somewhere, because it just sounds to me like an issue waiting to happen.
```suggestion
	if(target.wear_id?.get_gun_permit_iconstate() == "hud_permit")
```

Reply:
> I feel like `"hud_permit"` should be a define somewhere, because it just sounds to me like an issue waiting to happen.

Ahh, can they go in /code/_DEFINES/~skyrat_defines/gun.dm? I assume after that I just change any code with hud_permit/hudfan_no to the defines?

Also, what kind of issues can it cause? Asking because I'm fairly new here.

Reply:
I wrote two and put them in there, then I renamed the strings in data_huds.dm to the new defines.
hudfan_no is also used for clown/mime fan huds in that same file, but I'll leave that be since it's /tg/.